dali/base/dacoven.cpp

```diff
@@ -1009,7 +1028,7 @@ DALI_UID getGlobalUniqueIds(unsigned num,SocketEndpoint *_foreignnode)
     if (foreignnode.port==0)
```

Comment:
Could it ever be that this routine is called but the _foreignnode argument supplied is null or 0.0.0.0 ?